.gitmodules

```
@@ -6,3 +6,7 @@
 	path = lib/BAMIntervalTree
 	url = https://github.com/joshuak94/BAMIntervalTree.git
 	branch = master
+[submodule "b.i.o."]
+	path = lib/b.i.o.
+	url = https://github.com/h-2/b.i.o..git
```

```suggestion
	url = ../../h-2/b.i.o..git
```

I recently learned that you can use relative paths here, too. This respects choice to clone with ssh or https and has some other benefits. But it is also not really important 🙃